src/main/webapp/script.js

```diff
@@ -19,13 +19,18 @@ google.charts.load('current', {
 });
 
 function passQuery() {
+  document.getElementById('map-title').innerHTML = '';
   document.getElementById('result').style.display = 'block';
   const information = document.getElementById('more-info');
   information.innerText = 'Please wait. Loading...';
   const query = new FormData(document.getElementById('query-form'));
   const personType = query.get('person-type');
   const action = query.get('action');
   const location = query.get('location');
+  const region =
```

Comment:
Nit but I might add an empty line here, and maybe one more somewhere above, for increased readability

Reply:
Done!